sailor/utils/utils.py

```diff
@@ -1,6 +1,9 @@
 """Other utility functions that don't fit into any of the specific modules."""
 
 from collections.abc import Iterable
+import logging
+from logging import WARNING
```

Comment:
is that import necessary? why not just do `logging.WARNING` in line 33?

Reply:
import removed, instead `logging.WARNING`is used in line 33